cmd/geth/snapshot.go

```diff
@@ -40,6 +47,9 @@ var (
 
 	// emptyCode is the known hash of the empty EVM bytecode.
 	emptyCode = crypto.Keccak256(nil)
+
+	//the layer of tries trees that keep in memory
+	TriesInMemory = 128
```

Comment:
`TriesInMemory` is not a fixed value,  default value is 128, but some node use other values.

Reply:
Done, introduced in the user CLI command.